LICENSES/MIT.txt

```diff
@@ -0,0 +1,20 @@
+Open Source License
```

Comment:
I wonder where this "Open Source License" thing comes from (originally). Anyway, I think it's simpler to use this license https://github.com/tqtezos/baseDAO/blob/master/LICENSE, it seems to differ only in the header.

Also it seems more proper to name this file `LicenseRef-MIT-TQ.txt` because we may have another MIT license later (and that's how we name MIT license files usually).